### PR TITLE
user info component

### LINKED ISSUE #30: User Info component
Add it to Header

Should Have a User icon and logged in username next to it.

### PR DESCRIPTION
**Pull request by ololokiras**
#### What's this PR do?
show user info component in header
#### Where should the reviewer start?
src/views/UserInfo.vue
#### How should this be manually tested?
See account icon, account name and arrow in Header
#### Any background context you want to provide?
Nope
#### What are the relevant tickets?
closes #30 
#### Screenshots (if appropriate)
image
#### Questions:
How import icons in combine style?

### CODE REVIEW

**Review by roma219 — DISMISSED** [review]
👍

**Comment by roma219** [comment]
@dls-cipher @hisinbox merge please

**Comment by hisinbox** [comment]
This branch has conflicts that must be resolved

**Comment by ololokiras** [comment]
@hisinbox fixed

File: src/views/Header.vue [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
-  components: { ConnectionIndicator }
+  components: { UserInfo, ConnectionIndicator }
 }
 </script>
 
 <style >
   .header {
     @apply p-2;
     border: 1px solid black;
-    display: flex;
```
Comment (roma219) [inline src/views/Header.vue]: why delete? this is made to align header content

File: src/views/UserInfo.vue [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<template>
+  <div class="singleline">
+    <svgicon name="userIcon"/>
+    <p>{{ getName }}</p>
```
Comment (roma219) [inline src/views/UserInfo.vue]: Can do it simplier `{{ userName || 'default' }}` - and you won't need computed for this

**Review by roma219 — CHANGES_REQUESTED** [review]
Needs a couple smallfixes, otherwise good

**Review by roma219 — APPROVED** [review]
👍

**Comment by hisinbox** [comment]
looks good. will update design in due course

**Pull request merged**